tests/e2e/utils.py

```diff
@@ -11,7 +11,7 @@
 
 BASE_DATA_PATH = Path(f"{__file__}").parent / Path("files")
 DEFAULT_SOURCES = Path(f"{__file__}").parent / Path("../sources")
-DEFAULT_INVENTORY = BASE_DATA_PATH / "inventories/default_inventory.yml"
+DEFAULT_INVENTORY = BASE_DATA_PATH / "inventories/default_inventory.ini"
```

Comment:
Why not instead of overriding the current inventory, we create a new one so we are able to test both?

Reply:
@Alex-Izquierdo The debug action tries to do a pretty print of all its arguments which introduces new line characters.
So if the data is in yaml format we used to store this as a dictionary. When I read it as raw data it introduces new line characters and breaks this comparison.

https://github.com/ansible/ansible-rulebook/blob/70eac893fff643236c3db4334afc7ee07ac860e6/tests/e2e/test_actions.py#L100

https://github.com/ansible/ansible-rulebook/blob/70eac893fff643236c3db4334afc7ee07ac860e6/ansible_rulebook/builtin.py#L95

I struggled with keeping the yaml inventory file and tried doing all kinds of pprint tricks to get that test to pass.
This is the only test that matches the debug output.
I gave up and switched to INI file which is easier to match.

Reply:
yes, for this test you can use the ini, but leave the yaml for the rest, so we are able to test both. It's just a matter to define the inventory in this test, instead of the default